app/models/concerns/profile_tailoring.rb

```diff
@@ -17,7 +17,7 @@ def tailored?
   end
 
   def added_rules
-    rules - parent_profile.rules
+    rules.order(:precedence) - parent_profile.rules
```

Comment:
We might add this as a default order within APIs.

Reply:
In the followup issue that deals with remediations.